projects/stader/index.js

```diff
@@ -63,32 +72,54 @@ module.exports = {
     tvl: maticTvl
   }, */
   fantom: {
-    tvl: ftmTvl
+    tvl: ftmTvl,
   },
   terra2: {
-    tvl: terra2Tvl
+    tvl: terra2Tvl,
   },
   bsc: {
-    tvl: bscTvl
+    tvl: bscTvl,
   },
   near: {
-    tvl: nearTvl
+    tvl: nearTvl,
   },
   ethereum: {
     tvl: async (_, _1, _2, { api }) => {
+      const res = await getData();
+      const nodeOperatorRegistry = "0x4f4bfa0861f62309934a5551e0b2541ee82fdcf1";
+      const nodeOperatorCount = await api.call({
+        abi: "uint256:totalActiveValidatorCount",
+        target: nodeOperatorRegistry,
+      });
+
+      const SDCollateralPoolAddress =
+        "0x7Af4730cc8EbAd1a050dcad5c03c33D2793EE91f";
+      const SDTokenAddress = "0x30D20208d987713f46DFD34EF128Bb16C404D10f";
+
+      const sdBalance = await api.call({
```

Comment:
sorry, we cant count SD as tvl, it is protocols own token

Reply:
@g1nt0ki This is the bonded SD (bonded by Node operators for each validator held along with 4 ETH) and not the SD held by a Stader account or an EOA.

Reply:
sorry, but rules are rules. https://docs.llama.fi/list-your-project/what-to-include-as-tvl we could include this value under staking?

Reply:
@g1nt0ki the rules state if my tokens are locked on another protocol. Not sure if you would consider Stader's ETHX as another protocol.
This works the same as Rocktpool where they. have added their own RPL token to the TVL. Please check
https://github.com/DefiLlama/DefiLlama-Adapters/blob/e6748034efe43631b96b60bedea0ecafa29583fa/projects/rocketpool/index.js#L4

Reply:
@rahuljaguste my bad, both of you are right.